hoomd/write/dcd.py

```diff
@@ -0,0 +1,83 @@
+import hoomd
+from hoomd import _hoomd
+from hoomd.filter import ParticleFilter, All
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.operation import Writer
+
+
+class DCD(Writer):
+    """Writes simulation snapshots in the DCD format.
+
+    Args:
+        filename (str): File name to write.
+        trigger (hoomd.trigger.Periodic): Select the timesteps to write.
+        filter (hoomd.filter.ParticleFilter): Select the particles to write.
+            Defaults to `hoomd.filter.All`.
+        overwrite (bool): When False, (the default) an existing DCD file will be appended to. When True, an existing DCD
+                          file *filename* will be overwritten.
+        unwrap_full (bool): When False, (the default) particle coordinates are always written inside the simulation box.
+                            When True, particles will be unwrapped into their current box image before writing to the dcd file.
+        unwrap_rigid (bool): When False, (the default) individual particles are written inside the simulation box which
+               breaks up rigid bodies near box boundaries. When True, particles belonging to the same rigid body will be
+               unwrapped so that the body is continuous. The center of mass of the body remains in the simulation box, but
+               some particles may be written just outside it. *unwrap_rigid* is ignored when *unwrap_full* is True.
+        angle_z (bool): When True, the particle orientation angle is written to the z component (only useful for 2D simulations)
+
+    On each timestep where `DCD` triggers, it writes the simulation snapshot to the
+    specified file in the DCD file format. DCD only stores particle positions,
+    in distance units.
+
+    Examples::
+
+        hoomd.write.DCD("trajectory.dcd", hoomd.trigger.Periodic(1000))
+        dcd = hoomd.write.DCD(filename="data/dump.dcd", trigger=hoomd.trigger.Periodic(100, 10))
+
+    Warning:
+        When you use dump.dcd to append to an existing dcd file:
```

Comment:
```suggestion
        When you use `hoomd.writer.DCD` to append to an existing dcd file:
```
This needs to be changed in a few places.